modules/virt-creating-layer2-nad-cli.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-creating-layer2-nad-cli_{context}"]                                   
+= Creating a NAD for flat layer 2 topology using the CLI
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the layer 2 overlay network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` object. The following example configures a cluster-wide overlay without a subnet defined. This means that the logical switch implementing the network only provides layer 2 communication. You must define a static IP address for the virtual machine.
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l2-network
+  namespace: default
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1", <1>
+            "name": "l2-network", <2>
+            "type": "ovn-k8s-cni-overlay", <3>
+            "topology":"layer2", <4>
+            "mtu": 1300, <5>
+            "netAttachDefName": "default/l2-network" <6>
+    }
+----
+<1> The CNI specification version. This value must be less than `1.0.0`.
+<2> The name of the network. This network is not namespaced. For example, you can have a network named `l2-network` referenced from two different `NetworkAttachmentDefinition` objects that exist in two different namespaces. This feature is useful to connect VMs in different namespaces.
+<3> The name of the CNI plug-in to be configured. The required value is `ovn-k8s-cni-overlay`.
+<4> The topological configuration for the network. The required value is `layer2`.
+<5> Optional: The maximum transmission unit (MTU) value. The default value is automatically set by the kernel.
+<6> The value of the `namespace` and `name` fields in the `metadata` stanza of the `NetworkAttachmentDefinition` object.
+
+
+. Apply the manifest:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
```

Comment:
We use the actual name in the rest of our docs: https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge

Reply:
It is not consistently the case in the rest of our docs. For example: https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-using-dpdk-with-sriov.html#virt-configuring-vm-dpdk_virt-using-dpdk-with-sriov
Because the file name is a user-supplied value, I would prefer to keep this as is.

Reply:
Gotcha, all good then